### DISCUSSION TITLE
Mi az a nagy mínusz a tárgynapon!

### DISCUSSION THREAD

**Discussion by ferenchalmai** [ZsBT/hass-w1000-portal#18] [Q&A]
Sziasztok!
Még nem teljesen látom át, hogyan működik a beolvasó rész, de valami nem teljesen kerek benne. Mindenki azt írja, hogy csak az előző napi adatok működnek. Ez nem így van, a tárgy napi adatok is szépen jönnek le óránként, csak valamiért van benne egy nagy fals minuszos érték és ezért az elviszi a grafikont. De nem teljesen világos, hony az a szám honnan jön, mert nem az előző napi záró, látszólag nem függ össze semmivel. És ez a minuszos érték halad előre a nap folyamán és kúsznak utána a nulla közeli óránkénti adatok, kis vonalak képében. Mi okozhatja azt a fals számot? És az érdekesség, hogy ha az aktuális W1000.import értékét megnézem az mindig pozitív!
Üdv
Feró

![image](https://user-images.githubusercontent.com/116803060/222074027-10cbe639-37ab-4cd0-884d-6a7a4bb18654.png)

**Comment by ZsBT** [comment]
szia! igen ez egyelőre ilyen sajnos, azaz pont az utolsó napot csúnyítja el a szenzor.
ennek javításához nagy átalakítás kell, ráadásul mindenkinek át is kell írni a konfigját és az EON portálon is átrendezni a riportokat.
remélhetőleg a következő hetekben eljutok odáig, hogy legalább egy teszt kiadható legyen.

**Reply by Rannius** [reply]
Szia, van esetleg valami előrejutás ezzel kapcsolatban? :)
Köszi a választ.

**Comment by ZsBT** [comment]
Nem sok, de jogos, már régóta ígérgetem hogy befejezem végre! Igyekszem a
közeljövőben fixálni ezt.

**Reply by Rannius** [reply]
Semmi siettetés :D csak nagyon várom ;D
Köszi a munkádat!

**Reply by Rannius** [reply]
Szia, van valami fejlemény esetleg? *.*

**Reply by ZsBT** [reply]
Igen, már tesztelek egy verziót magamnál.

**Comment by ferenchalmai** [comment]
Sziasztok!

Nekem a W1000 portál megbízhatatlan volt, így váltottam P1 port olvasóra. Atom stabil.
Az EON levélben küldi a napi fogyasztást , termelést. Havonta többször 0 amit küld.
Nem megbízható.

De azért szép munka volt az eddigi is, csak nem tudom perspektívikus-e még a fejlesztése. Mert a távleolvasós órák nagyrészén van P1 port is.

Nekem ennek az előző verziója van:
https://www.zuidwijk.com/product/slimmelezer-plus/

És sokan használják még ezt is, megelégedéssel:
https://www.homewizard.com/p1-meter/
Ha szerencséd van akkor nem kell külön táp hozzájuk.

Üdv

Halmai Ferenc

**Reply by Rannius** [reply]
Az a baj, hogy nekem pl rettentő messze van a villanyóra a háztól. nem tudok oda internetet biztosítani.

**Reply by ZsBT** [reply]
@ferenchalmai a P1 porthoz hogy fértél hozzá? Nálam a duplán dobozolt-plombált utólag nem hozzáférhető, még nőgyógyász se érné el.   HUP-os fórum szerint a bontás-újradobozolás és plombálás horror összeg lenne. Ahogy néztem, egy háromfázisos wifis lakatfogó is olcsóbb lenne annál...

**Comment by ferenchalmai** [comment]
Hali!

Nálam nyitható a szekrény több részletben is és csak az óra van lezárva. A horror összeg olyan 30 körüli.

Lakatfogóra én most ezt láttam, óccó, kicsi:

https://www.youtube.com/watch?v=n_A992Bu9ik

https://www.zemismart.com/collections/sensor

Van 3 fázisra is és akár konnektorba betehető verzió is. Ugye sokszor felmerült melyik okos konnektor mekkora terhelést bír, na itt ez nem kérdés!
63A-ig mér. Igaz nem lehet vele lekapcsolni készüléket, de ha nem is ez a cél akkor nem rossz. És kétirányt mér, könnyen fel lehet rakni a bejövő ágra is .
WIFI és Zigbee verzió is van.


Üdv

**Comment by Rannius** [comment]
Szia ebbe lesz változás? Köszi a választ.

**Reply by ZsBT** [reply]
Szia! Lesz-lesz, a saját gépemen már tesztelek egy verziót ami ezt javítja. Eléggé újra kellett írni hozzá az integrációt, még recseg-ropog.
